<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gra_control</class>
 <widget class="QMainWindow" name="Gra_control">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="zbite_1_out">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>640</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>zbite1</string>
    </property>
   </widget>
   <widget class="QLabel" name="zbite_2_out">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>690</y>
      <width>640</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>zbite2</string>
    </property>
   </widget>
   <widget class="MyLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>640</width>
      <height>640</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>plansza</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="pole_konc_out">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>50</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="komunikat_out">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>130</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>660</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>polaczenie:</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>90</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pole startowe:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="pole_start_out">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>50</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>170</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="powrot_do_menu">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>660</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>wróć do menu</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>210</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Lista dotychczasowych ruchów:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lista_ruchow">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>230</y>
      <width>251</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pole koncowe</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>110</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>                                  Komunikat:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>190</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Aktualny gracz:</string>
    </property>
   </widget>
   <widget class="QLabel" name="aktualny_kolor_out">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>190</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>kolor_gracza</string>
    </property>
   </widget>
   <widget class="QLabel" name="connect_label">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>670</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>not_connected</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>0</y>
      <width>16</width>
      <height>751</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>10</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ostatni ruch:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zapisz_do_FEN_in">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>540</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Zapisz pozycję jako kod FEN.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="wczytaj_z_FEN_in">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>570</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Wczytaj pozycje z kodu FEN.</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLabel</class>
   <extends>QLabel</extends>
   <header>mylabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
